AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_dependency_tests2.py

```diff
@@ -44,53 +43,71 @@ def local_resources(request, workspace, ap_setup_fixture):
 @pytest.mark.parametrize("project", targetProjects)
 @pytest.mark.assetpipeline
 @pytest.mark.SUITE_periodic
-class TestsAssetProcessorBatch_DependenycyTests(object):
+class TestsAssetProcessorBatch_DependencyTests(object):
     """
     AssetProcessorBatch Dependency tests
     """
     @pytest.mark.test_case_id("C18108049")
     @pytest.mark.BAT
     @pytest.mark.assetpipeline
-    def test_WindowsMacPlatforms_SchemaGem_BatchLoadsSchemaFromGem(
-            self, asset_processor, workspace):
+    def test_WindowsMacPlatforms_SchemaGem_BatchLoadsSchema(
```

Comment:
the naming seems to indicate "test_WindowsMacPlatforms" that it is only for Windows and Mac... but this is supposed to fix the Linux issues. Maybe this test should be renamed to "test_SchemaGem_BatchLoadsSchema" ?

Reply:
Good point, I've removed windows & mac from the test name